src/errors/RequestResponseError.ts

```diff
@@ -4,30 +4,32 @@
 
 import { ErrorResponse } from "../api/RestAPI";
 
-const serializeErrorResponse = ({ code, httpCode, errors }: Omit<ErrorResponse, "status">): string =>
-    `Code: ${code}, HttpCode: ${httpCode}, Errors: ${errors
+type ErrorRequest = Omit<ErrorResponse, "status">;
+
+const serializeErrorResponse = ({ code, httpCode, errors }: ErrorRequest): string =>
+    `Code: ${code}, HttpCode: ${httpCode}, Errors: ${(errors || [])
```

Comment:
This is the only logic change `errors` -> `(errors || [])`. `errors` can be undefined in some cases such as the gateway credentials.